src/main/java/ch/njol/skript/effects/EffHealth.java

```diff
@@ -34,42 +17,46 @@
 import ch.njol.util.Math2;
 import org.bukkit.entity.Damageable;
 import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityDamageEvent.DamageCause;
 import org.bukkit.inventory.ItemStack;
-import org.eclipse.jdt.annotation.Nullable;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
 
 @Name("Damage/Heal/Repair")
-@Description("Damage/Heal/Repair an entity, or item.")
+@Description("Damage, heal, or repair an entity or item.")
 @Examples({
 	"damage player by 5 hearts",
+	"damage player by 3 hearts with fake cause fall",
 	"heal the player",
 	"repair tool of player"
 })
 @Since("1.0")
```

Comment:
While I'm at this let's see about adding a `@RequiredPlugins` annotation pointing towards 1.20.4+